sdk/digitaltwins/tests.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+
+jobs:
+- template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: digitaltwins
+    Location: southcentralus
+    AzureCloudSubscriptionId: $(azure-subscription-preview-id)
```

Comment:
FYI @danieljurek you should be able to help fix the subscription stuff now in master.